client/payment-details/dispute-details/test/index.test.tsx

```diff
@@ -52,11 +52,13 @@ global.wcpaySettings = {
 	},
 };
 
-interface ChargeWithDisputeRequired extends Charge {
-	dispute: Dispute;
+interface ChargeWithDisputeEvidenceDetails extends Charge {
+	dispute: Dispute & {
+		evidence_details: EvidenceDetails;
+	};
```

Comment:
This modification makes it easier to override `charge.dispute.evidence_details` within this file without TS warning that `evidence_details` may be `undefined` – we're setting a value in `getBaseCharge()`.